packages/liquid-html-parser/grammar/liquid-html.ohm

```diff
@@ -403,7 +403,8 @@ LiquidDoc <: Helpers {
   paramNode = "@param" strictSpace* paramType? strictSpace* paramName (strictSpace* "-")? strictSpace* paramDescription
   paramType = "{" strictSpace* paramTypeContent strictSpace* "}"
   paramTypeContent = anyExceptStar<("}"| strictSpace)>
-  paramName = identifierCharacter+
+  paramName =  "["? strictSpace* paramNameContent strictSpace* "]"?
```

Comment:
Since only having a single square bracket isn't valid I wonder if we should change to something like this:

```ohm
paramNode = "@param" strictSpace* paramType? strictSpace* (optionalParam | paramName) (strictSpace* "-")? strictSpace* paramDescription
optionalParam = "[" strictSpace* paramName strictSpace* "]"
```

Then we either match on `optionalParam` _or_ `paramName` and there's no ambiguity.

Reply:
Modified the approach to go in this direction. This helps resolve some other downstream issues I was having deciding on behaviour, so 💯 agree